sonic_platform_base/sonic_xcvr/api/public/cmis.py

```diff
@@ -2478,4 +2478,13 @@ def get_error_description(self):
 
         return None
 
+    def apply_app0_configuration(self):
+        '''
+        This function applies App0 configuratino - in case some lanes are de-activated
+        '''
+        for lane in range(self.NUM_CHANNELS):
+            addr = "{}_{}_{}".format(consts.STAGED_CTRL_APSEL_FIELD, 0, lane + 1)
```

Comment:
In my opinion F strings are a better then format:
```suggestion
            addr = "{consts.STAGED_CTRL_APSEL_FIELD}_0_{lane + 1}"
```
But there isn't a clear cut reason to avoid using str.format